app/controllers/Export.scala

```diff
@@ -33,14 +33,14 @@ final class Export(env: Env) extends LilaController(env) {
     }
   }
 
-  def gif(id: String) = Open { implicit ctx =>
+  def gif(id: String, color: String) = Open { implicit ctx =>
     OnlyHumansAndFacebookOrTwitter {
       ExportRateLimitGlobal("-", msg = HTTPRequest.lastRemoteAddress(ctx.req).value) {
-        OptionFuResult(env.game.gameRepo gameWithInitialFen id) {
-          case (game, initialFen) =>
-            env.game.gifExport.fromGame(game, initialFen) map
+        OptionFuResult(env.game.gameRepo povWithInitialFen(id, color)) {
```

Comment:
Not great that I'm passing along raw strings here and making a new method `povWithInitialFen`, but I don't think I can instantiate `Pov` or `Color` in this file?

Reply:
Yes you can. Controllers are at the top of the tree, they can use all the modules. `lila.game.Pov` is available.

Reply:
Ok probably fine either way. Thanks!